internal/controller/grpc_logger.go

```diff
@@ -78,7 +83,7 @@ func (b *BufferedGrpcWriteSyncer) flush() {
 		lostLogEntry := &zapcore.Entry{
 			Level:   zap.ErrorLevel,
 			Time:    time.Now(),
-			Message: "Lost logs due to buffer overflow",
+			Message: `{"msg":"Lost logs due to buffer overflow"}`,
```

Comment:
The purpose of this PR is confusing to me because I am missing context.

But given that the purpose is correct, can we expose an easier to use API? Keep the data (log message) separate from the metadata. Like so:
```suggestion
			Message: iloLogMessage("Lost logs due to buffer overflow").AsString(),
```

Or like so:
```
iloLogMessage{
    Level:   zap.ErrorLevel,
    Message: "Lost logs due to buffer overflow",
}.AsZapEntry()
```

----

Maybe this is not the type of thing that needs to get written a lot tho,,, and it's fine to leave as-is

Reply:
This is not the final solution, as I don't think it's working.
This PR is not yet ready for review.
Please hold off for a bit...